node/node.go

```diff
@@ -59,7 +61,7 @@ func HashId(key string) uint32 {
 // Load nodes config file
 func LoadNodesConfig(config_file string) NodesConfig {
 	file, _ := ioutil.ReadFile(config_file)
-	nodes_config := NodesConfig{}
+	nodes_config := NodesConfig{EnableClientAuth: true, EnableHttps: true}
```

Comment:
Quick question, why are `EnableClientAuth` and `EnableHttps` hard coded to `true` here? Shouldn't these values come from the config file or command line flag?

Reply:
Hi @malwaredllc, if unspecified, these will be the default values, meaning if the config have the fields to be absent, it will have https and mTLS as enabled by default